src/main/kotlin/com/openlattice/conductor/rpc/ConductorConfiguration.kt

```diff
@@ -0,0 +1,36 @@
+package com.openlattice.conductor.rpc
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.kryptnostic.rhizome.configuration.Configuration
+import com.kryptnostic.rhizome.configuration.ConfigurationKey
+import com.kryptnostic.rhizome.configuration.SimpleConfigurationKey
+import com.kryptnostic.rhizome.configuration.annotation.ReloadableConfiguration
+
+
+private val REPORT_EMAIL_ADDRESS_FIELD = "reportEmailAddress"
+private val SEARCH_CONFIGURATION_FIELD = "searchConfiguration"
+private val key = SimpleConfigurationKey("conductor.yaml")
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+@ReloadableConfiguration(uri = "conductor.yaml")
+class ConductorConfiguration(
+        @JsonProperty("reportEmailAddress") val reportEmailAddress: String,
```

Comment:
should this be using the constants defined above? also do we need to update existing conductor.yamls for the new connection param?

Reply:
I think changed this to optional. If you do specify, it will add all the users to the global org. I think we found a better solution than this.

Reply:
Also for kotlin data files defining constants inline for non-reused fields is perfectly fine. You don't have to worry about synchronization with getters and setters. Obviously, for re-used fields they should be in SerializatonConstants.

Reply:
Oh my bad -- I thought I saw unused constants defined for these values right above the constructor. I must have been looking at the java version or something.